README.md

```diff
@@ -1,129 +1,108 @@
+probably want to fork this repo, make it your own, and maybe contribute some
+tools back if they are more generally useful.
 
-Install [Miniconda3](https://docs.conda.io/en/latest/miniconda.html#linux-installers).
+To get started, fork and clone the repo.
 
-    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-    bash Miniconda3-latest-Linux-x86_64.sh
+Next, you will want to crate a configuration for the infrastructure that
+you would like to run.  To do so, create a file `terraform/main.tf` that
+describes your development infrastructure.  For example,
+my `main.tf` looks like this:
 
-Install [Mamba](https://github.com/mamba-org/mamba) to speed up environment building
+```hcl
+module "nitro_dev" {
+  source = "./modules/nitro"
+  instance_name = "dlm-terraform-test"
+  key_pair_name = "dlm-dev"
+}
 
-    conda install mamba -n base -c conda-forge
+output "nitro_dev_instance_ip" {
+  description = "The public IP address of the instance"
+  value       = module.nitro_dev.instance_public_ip
+}
+```
 
-From the base environment, create and activate (update) a Python environment
-specified in `environment.yaml`
+For the "key_pair_name", you will need to have a key pair already created in
+AWS and the private key on your local machine.  Below, we will refer to the
+location of this key on your local machine as `<path-to-your-pem-file>`.
 
-    mamba env update -n bky-ned -f environment.yaml
-    conda activate bky-ned
+You will likely want to create an output with the ip address of the
+instance that is created so that you can use it later in ansible playbooks.
 
-Install all the python dependencies
+Let's create the infrastructure.
 
-    poetry install
+```bash
+cd terraform
+terraform init
+terraform plan -out tfplan
+terraform apply tfplan
+```
 
-Make sure that everything is set up properly for development by running:
+Upon successful completion of the `apply` you will see the ip address of the
+instance that was created. You can copy it for later or put it into an
+environment variable like so:
 
-    make test
+```bash
+ip=$(terraform output nitro_dev_instance_ip)
+```
 
-When you want to clean up run:
+Also, if you hop over to the aws console, you will see that your instance is
+deployed. You can also get the ip address from the console. Either way, let's
+clean up the plan we used and move on to the next step.
 
-    conda deactivate
-    conda remove -n sequencer --all
-    make veryclean
+```bash
+rm tfplan
+cd ..
+```
 
-While you can run unit tests without configuring the NED, you will need to
-[Configure NED](#configuring-ned) to run `make test-live` (live tests) or use the
-application.
+Let's use our `init-nitro` playbook to minimally set up a system.
 
-<a name="configuring-ned"></a>
-###  Configuring NED
+```bash
+ansible-playbook \
+    -i ansible/inventories/dev.yml \
+    -e ip=$ip \
+    --key-file <path-to-your-pem-file> \
+    ./ansible/playbooks/init-nitro.yml
+```
```

Comment:
it seems like if we have $ip already in the shell, it might make sense from a readme perspective to encourage the user to store the <path-to-your-pem-file> value as a variable as well so this could be a copy paste command